src/importexport/mei/internal/meiimporter.cpp

```diff
@@ -1254,6 +1254,16 @@ bool MeiImporter::readSectionElements(pugi::xml_node parentNode)
 {
     bool success = true;
 
+    libmei::Section meiSection;
+    meiSection.Read(parentNode);
+
+    if (m_score->measures()->size() && meiSection.HasRestart() && meiSection.GetRestart() == libmei::BOOLEAN_true) {
```

Comment:
Last comment, sorry :) It's better to use !empty() instead of implicitly casting size() to bool